### DISCUSSION TITLE
Shared Root Domain

### DISCUSSION THREAD

**Discussion by rehatkathuria** [mastodon/mastodon#19720] [Q&A] [answered]
I have a domain that I would like to for a username whilst also serving my `next.js` website. Is this feasible?

Example username: `toot@example.com`
The domain I want to continue serving my website through: `example.com`

**Comment by puyoxyz** [comment, marked as answer]
Yes, with [WEB_DOMAIN config option](https://docs.joinmastodon.org/admin/config/)
